### DISCUSSION TITLE
Hexo NexT 魔改系列之三 ── 评论篇

### DISCUSSION THREAD

**Discussion by giscus** [wangjiezhe/wangjiezhe.github.io#28] [Announcements]
# Hexo NexT 魔改系列之三 ── 评论篇

为NexT主题添加评论功能

https://blog.wangjiezhe.com/posts/2018-10-29-Hexo-NexT-3/

**Comment by wangjiezhe** [comment]
Giscus 评论测试

**Comment by wangjiezhe** [comment]
自建 Giscus 评论
